man/systemd-sysext.xml

```diff
@@ -205,6 +212,51 @@
     to tie the most frequently configured options to runtime updateable flags that can be changed without a
     system reboot. This will help reduce servicing times when there is a need for changing the OS configuration.</para></refsect1>
 
+  <refsect1>
+    <title>Mutability</title>
```

Comment:
This paragraph should include the version snippet, not sure if the <xi:include> works in this context, if it doesn't please write by hand that it was introduced in v256

Reply:
`xi:include` seems to work - tests succeed locally, no error reported.